### PR TITLE
home page ideas

### PR DESCRIPTION
**Pull request by vojtaholik**
[➭ preview](https://egghead-io-nextjs-lf9r1xcyj.vercel.app/home)

home@2x

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2F14bWswbeWGzYEo%2Fgiphy.gif&token=adventure" data-gitmeme-token="adventure"></a>

### CODE REVIEW

File: src/components/home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,693 @@
...
+import Link from 'next/link'
+import Image from 'next/image'
+import {map, get, find} from 'lodash'
+import Textfit from 'react-textfit'
+import Markdown from 'react-markdown'
+
+type HomeProps = {}
+
+const Home: FunctionComponent<HomeProps> = () => {
+  const data = [
```
Comment (vojtaholik) [inline src/components/home/index.tsx:13]: this is naive at best, but hopefully fine for initial pass? I haven't put much thought into whole authoring experience...

File: src/components/home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,693 @@
...
+  const schedule: any = find(data, {id: 'schedule'})
+  const featured: any = get(find(data, {id: 'featured'}), 'resources', {})
+  const devEssentials: any = find(data, {id: 'devEssentials'})
+  const stateManagement: any = find(data, {
+    id: 'stateManagement',
+  })
+  const sideProject: any = find(data, {id: 'sideProject'})
+  const mdxConf: any = find(data, {id: 'mdxConf'})
+  const topics: any = find(data, {id: 'topics'})
+  const swag: any = find(data, {id: 'swag'})
```
Comment (vojtaholik) [inline src/components/home/index.tsx:345]: grabbing objects by id. they all follow same general format.

File: tailwind.config.js [hunk 3]
```diff
@@ -86,12 +86,15 @@ module.exports = {
...
+      },
     },
   },
   variants: {
     margin: ['responsive', 'first', 'last'],
     padding: ['responsive', 'first', 'last'],
     scale: ['hover'],
   },
-  plugins: [require('@tailwindcss/typography', require('@tailwindcss/ui'))],
+  plugins: [require('@tailwindcss/typography')],
```
Comment (vojtaholik) [inline tailwind.config.js:99]: tailwind v2 already comes with all `@tailwindcss/ui` had to offer.

File: src/components/home/index.tsx (hunk 1 above)
Comment (joelhooks) [reply src/components/home/index.tsx]: i think this “brute force data glob” is a perfect place to start and will let us make a more considered authoring experience in the future. I dig it.

**Review by joelhooks — APPROVED** [review]

File: src/components/home/index.tsx (hunk 1 above)
Comment (vojtaholik) [reply src/components/home/index.tsx]: I also wrote this tiny transform function that takes a url and returns data in a format that is expected here. 
saved me a little bit of time: https://codesandbox.io/s/quick-data-transformer-brjd3

**Pull request merged**